account_asset_management/models/account_asset_line.py

```diff
@@ -0,0 +1,331 @@
+# Copyright 2009-2018 Noviat
+# Copyright 2021 Tecnativa - João Marques
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class AccountAssetLine(models.Model):
+    _name = "account.asset.line"
+    _description = "Asset depreciation table line"
+    _order = "type, line_date"
+    _check_company_auto = True
+
+    name = fields.Char(string="Depreciation Name", size=64, readonly=True)
+    asset_id = fields.Many2one(
+        comodel_name="account.asset",
+        string="Asset",
+        required=True,
+        ondelete="cascade",
+        check_company=True,
+        index=True,
+    )
+    previous_id = fields.Many2one(
+        comodel_name="account.asset.line",
+        string="Previous Depreciation Line",
+        readonly=True,
+    )
+    parent_state = fields.Selection(
+        related="asset_id.state", string="State of Asset", readonly=True
+    )
+    depreciation_base = fields.Float(
+        related="asset_id.depreciation_base", string="Depreciation Base", readonly=True
+    )
+    amount = fields.Float(digits="Account", required=True)
```

Comment:
i think we can not require Float field.

Reply:
I think you are right. But this is being addressed here https://github.com/OCA/account-financial-tools/pull/1513 . I think it is better to consider this as a migration issue. And add the PR change to v15 and v16 in another PR.

Reply:
hi i add pr https://github.com/OCA/account-financial-tools/pull/1513 and now it ok